applications/registration-react/src/components/reg-form-schema-accessRights/form.jsx

```diff
@@ -22,7 +22,7 @@ export const renderLegalBasisFields = (item, index, fields, customProps) => (
   <div className="d-flex mb-2" key={index}>
     <div className="w-50">
       <Field
-        name={`${item}.prefLabel.nb`}
+        name={`${item}.prefLabel[${localization.getLanguage()}]`}
```

Comment:
hvis dette skulle bugge opp et javascript string, da er en bug her. De regner ut som 

`itemValue.prefLabel[nb]`. Je tenke at to ville ha `itemValue.prefLabel["nb"]`

Reply:
Rettet til mønster name={`${item}.prefLabel.${localization.getLanguage()}`} alle steder